asadiqbal08/SOL-1455: UI Enhancements

asadiqbal08 opened the pull request:
@mattdrayer kindly take a look.

cc: @griffresch @ziafazal

asadiqbal08 commented:
jenkins run all

asadiqbal08 commented:
jenkins run quality

mattdrayer commented:
@asadiqbal08 LGTM code-wise -- you may need to make another small tweak based on @griffresch's comments in SOL-1455 -- so :+1: outside of that modification, plus one more review from @ziafazal and we should be all set

asadiqbal08 commented:
@mattdrayer I have rebased and updated the PR Now i am using the Pre-existed css class as defined under .btn-blue and I have measured height/weight with page ruler in Chrome. 
Kindly let me know if anything else is required in it. 

Thanks

asadiqbal08 commented:
jenkins run all

asadiqbal08 commented:
jenkins run lettuce

asadiqbal08 commented:
@ziafazal PR is ready for review. thanks !

ziafazal commented:
@asadiqbal08 LGTM :+1:

mattdrayer commented:
Great, two thumbs, merging!
